integration_test/scripts/publish-images.sh

```diff
@@ -8,6 +8,8 @@ KMS_LOCATION=$6
 KMS_KEYRING=$7
 KMS_KEY=$8
 
+docker images | grep "healthcare-api-dicom-dicomweb-adapter-"
+
```

Comment:
Is this a debug line?

Reply:
yes, it was used for debugging and I will remove it

Reply:
Thanks